package.json

```diff
@@ -34,39 +34,39 @@
   "homepage": "http://eslint.org",
   "bugs": "https://github.com/eslint/eslint/issues/",
   "dependencies": {
-    "babel-code-frame": "^6.16.0",
+    "babel-code-frame": "^6.22.0",
     "chalk": "^1.1.3",
-    "concat-stream": "^1.5.2",
-    "debug": "^2.1.1",
+    "concat-stream": "^1.6.0",
+    "debug": "^2.6.3",
     "doctrine": "^2.0.0",
     "escope": "^3.6.0",
     "espree": "^3.4.0",
     "esquery": "^1.0.0",
     "estraverse": "^4.2.0",
     "esutils": "^2.0.2",
     "file-entry-cache": "^2.0.0",
-    "glob": "^7.0.3",
-    "globals": "^9.14.0",
-    "ignore": "^3.2.0",
+    "glob": "^7.1.1",
+    "globals": "^9.16.0",
+    "ignore": "^3.2.6",
     "imurmurhash": "^0.1.4",
-    "inquirer": "^0.12.0",
-    "is-my-json-valid": "^2.10.0",
+    "inquirer": "^3.0.6",
```

Comment:
I think inquirer switched from callbacks to Promises in v1.0.0, so I'm pretty sure this will require some changes.